R/pkg/inst/tests/testthat/jarTest.R

```diff
@@ -16,17 +16,17 @@
 #
 library(SparkR)
 
-sparkR.session()
+sc <- sparkR.session()
```

Comment:
we could change that in `print.jobj`? since we don't require specifying SparkSession in calls, we don't really need to return this from `sparkR.session()`?